src/components/shared/data-annotation/DataAnnotation.js

```diff
@@ -3,9 +3,34 @@
 import React from 'react';
 import fontSizes from '../../../constants/fontSizes';
 
-function DataAnnotation({ x, y, stroke, value, valueFormatter }) {
+function DataAnnotation({
```

Comment:
Ideally we could just pass `label={{position: 'top'}}` into components like `Bar` or `Line`, but because we made our own `DataAnnotation` component for more control, we have to handle this positioning logic.